src/app/globals.css

```diff
@@ -40,46 +40,20 @@ a {
   text-decoration: none;
 }
 
+
 @media (prefers-color-scheme: dark) {
   html {
     color-scheme: dark;
   }
 }
 
-.not-found-container {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  min-height: 100vh;
-  text-align: center;
-}
-
-.not-found-container h1 {
-  font-size: 5rem;
-  color: #B077FF;
+button{
+  font-size: 28px;
+  font-weight: 400;
+  line-height: 27.89px;
+  height: 48px;
+  width: 324px;
```

Comment:
the height and width of the buttons must be defined with the CSS tying to the active and inactive button components, not in globals.css. globals.css is a file for either global variables or CSS applied to all HTML elements

Reply:
also instead of width and height, to make a button a specific size you also want to use padding instead for responsiveness between devices